code/src/components/MovieDetails.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import { useParams, Link } from 'react-router-dom';
+
+import { API_KEY, imageURL } from 'utils/urls';
+
+export const MovieDetails = () => {
+  const [details, setDetails] = useState({})
+  const { movieId } = useParams()
+
+  useEffect(() => {
+    fetch(`https://api.themoviedb.org/3/movie/${movieId}?api_key=${API_KEY}&language=en-US`)
+      .then((response) => response.json())
+      .then((data) => {
+        setDetails(data)
+      });
+  }, [movieId]);
+
+  return (
+    <div className="details">
+      <img className="background-img" src={`${imageURL}/w1280/${details.backdrop_path}`} alt={details.id} />
```

Comment:
A suggestion here would be to use `{movie.title}` for the alt-text to make it more accessible.